testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -462,6 +463,30 @@ QUnit.module('general', {}, () => {
 
         themes.isMaterial = realIsMaterial;
     });
+
+    QUnit.test('label max-width should be changed with editor size (without option change)', function(assert) {
```

Comment:
I don't think it's a good idea:
1. test is not isolated. `resizeObserverSingleton.observe` is restored inside of a callback which can not be called
2. weird timeout (test can be unstable). Did u test it in different browsers?
3. Does this tests improves a test coverage? We have similar testcafe test